ocs_ci/ocs/node.py

```diff
@@ -1027,11 +1019,13 @@ def delete_and_create_osd_node_vsphere_upi_lso(osd_node_name, use_existing_node=
     assert is_completed, "ocs-osd-removal-job is not in status 'completed'"
     log.info("ocs-osd-removal-job completed successfully")
 
-    expected_num_of_deleted_pvs = 1
+    expected_num_of_deleted_pvs = [0, 1]
```

Comment:
shouldn't this be a dynamic variable that is being passed to the function?

Reply:
Do you mean that we should check inside the function how many PVs have been deleted and return `True` in case of success and `False` otherwise?

Reply:
either that or maybe the function can get the expected number of deleted PVs

Reply:
I think it will limit the function... cause the purpose of the function is to delete all the released PVs in a specific storage class.